File: 프로그래머스/0/120911. 문자열 정렬하기 （2）/문자열 정렬하기 （2）.java
import java.util.Arrays;

class Solution {  
    public String solution(String my_string) {
        String answer = "";
        my_string = my_string.toLowerCase();
        
        char[] list = my_string.toCharArray();
        Arrays.sort(list);
        
        answer = new String(list);
        
        return answer;
    }
}

File: 프로그래머스/0/120911. 문자열 정렬하기 （2）/문자열 정렬하기 （2）.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int* compare(const int* a, const int* b){
    return strcmp((char *)a, (char *)b);
}

char* solution(const char* my_string) {
    char* answer = (char*)malloc(strlen(my_string) * sizeof(char));
    strcpy(answer, my_string);
    
    for(int i=0;i<strlen(my_string);i++){
        if(my_string[i] >= 'A' && my_string[i] <= 'Z'){
            answer[i] -= 'A' - 'a';
        }
    }
    
    qsort(answer, strlen(my_string), sizeof(char), compare);

    return answer;
}